response.go

```diff
@@ -525,7 +525,7 @@ func NewNotFoundResponder(fn func(...any)) Responder {
 //	httpmock.NewStringResponse(200, httpmock.File("body.txt").String())
 func NewStringResponse(status int, body string) *http.Response {
 	return &http.Response{
-		Status:        strconv.Itoa(status),
+		Status:        fmt.Sprintf("%d %s", status, http.StatusText(status)),
```

Comment:
RFC2616 says that the status-code is a 3-digit integer. As the status comes from outside, perhaps should we use:
```suggestion
		Status:        fmt.Sprintf("%03d %s", status, http.StatusText(status)),
```
instead?
It doesn't protect us from negative and >999 statuses, but at least <100 ones conform to the RFC.
Note that http/httptest uses this format.

Reply:
This absolutely works for me. As far as implementation goes, I looked up Go's http, but http/httptest is just as valuable a source.